app/views/user_mailer/in_person_failed.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="s16">
+  <%= t('user_mailer.in_person_verified.greeting') %><br>
+  <%= t(
+        'user_mailer.in_person_failed.intro',
+        location: @presenter.location_name,
+        date: @presenter.formatted_verified_date,
+      ) %><br /><br />
+  <%= t('user_mailer.in_person_failed.body') %><br><br />
+  <strong><%= t('user_mailer.in_person_failed.verifying_identity') %></strong><br>
+  <ul class="usa-list">
+    <li><%= t('user_mailer.in_person_failed.verifying_step_not_expired') %></li>
+    <li><%= t('user_mailer.in_person_failed.verifying_step_proof_of_address') %></li>
+  </ul>
+</div>
+</p>
```

Comment:
Looks like a stray tag?

```suggestion
```

Reply:
fixed in e87a1979eeb6f903b2d66225e73508dc6b6c5654